src/spirit-view/spirit-view.html

```diff
@@ -96,6 +96,12 @@
                     }
                 }
                 return true;
+            },
+
+            _removeName: function() {
```

Comment:
The proposed solution won't work: you can tap in the paper-input, which clears the text, and then tap 'next'. This leads to no name being used.

Take a look at the [paper input demos](https://www.webcomponents.org/element/PolymerElements/paper-input/demo/demo/index.html). There's a feature they show using the "placeholder" attribute. Have you tried that? It sounds like it may be what we need.